tests/ludwig/augmentation/test_image_augmentation.py

```diff
@@ -0,0 +1,29 @@
+import pytest
```

Comment:
Let's add a test to this file that checks whether unexpected types supplied to the `augmentation` parameter cause a validation error.

Reply:
@ksbrar 
Took  a stab at the test.  https://github.com/ludwig-ai/ludwig/blob/d6ff7e3db9194866e477069481070a1f55aad98a/tests/ludwig/augmentation/test_augmentation_pipeline.py#L267-L289

If this looks reasonable, how extensive should the tests be for invalid parameters.  For example, `random_rotate` parameters needs to be a positive odd integer.  If we have to test for invalid values for this augmentation, I may need some help to define the schema conditions.

Reply:
This is a good one. I would add these cases:
* `None`
* Empty list `[]`
* Any random value like a string or something

Reply:
@ksbrar   Here are the test cases for the failure test
```
@pytest.mark.parametrize(
    "augmentation_pipeline_ops",
    [
        None,
        [
            {"type": "invalid_string"}
        ],
        [
            "random_horizontal_flip"
        ],
        "random_horizontal_flip",
        [
            {"type": "random_rotate", "degree": "45"},
        ],
    ],
)
```

All of these generate the `ValidationError`.

Interestingly enough, an empty list "`[]`" does not generate an error.  I'm still researching this.